generators/server/templates/src/main/java/package/config/_SecurityConfiguration.java

```diff
@@ -170,6 +188,10 @@ protected void configure(HttpSecurity http) throws Exception {
             .sessionManagement()
             .maximumSessions(32) // maximum number of concurrent sessions for one user
             .sessionRegistry(sessionRegistry)
+            .and().and()<% } %><% if (clusteredHttpSession == 'infinispan') { %>
+            .sessionManagement()
+            .maximumSessions(32) // maximum number of concurrent sessions for one user
+            .sessionRegistry(sessionRegistry())
             .and().and()<% } %>
```

Comment:
@jhipster/developers why do we use `and().and()` is it by mistake or was there a reason to do this? looks weird